packages/electron/README.md

```diff
@@ -0,0 +1,504 @@
+find them, you need to provide them manually via userAttributes attributes.
+
+There are several places where attributes can be added, modified or deleted.
+
+#### Attach attributes object to BacktraceClient
+
+It is possible to include an attributes object during [BacktraceClient](#backtraceclient) initialization. This list of
+attributes will be included with every error report, referred to as global attributes.
+
+```ts
+// Create an attributes object that can be modified throughout runtime
+const attributes: Record<string, unknown> = {
+    release: 'PROD',
+};
+
+// BacktraceClientOptions
+const options: BacktraceConfiguration = {
+    url: 'https://submit.backtrace.io/<universe>/<token>/json',
+
+    // Attach the attributes object
+    userAttributes: attributes,
+};
+
+// Initialize the client
+const client = BacktraceClient.initialize(options);
+```
+
+You can also include attributes that will be resolved when creating a report:
+
+```ts
+// BacktraceClientOptions
+const options: BacktraceConfiguration = {
+    url: 'https://submit.backtrace.io/<universe>/<token>/json',
+
+    // Attach the attributes object
+    userAttributes: () => ({
+        attribute: getAttributeValue(),
+    }),
+};
+
+// Initialize the client
+const client = BacktraceClient.initialize(options);
+```
+
+#### Add attributes during application runtime
+
+Global attributes can be set during the runtime once specific data has be loaded (e.g. a user has logged in).
+
+```ts
+const client = BacktraceClient.initialize(options);
+...
+
+client.addAttribute({
+    "clientID": "de6faf4d-d5b5-486c-9789-318f58a14476"
+})
+```
+
+You can also add attributes that will be resolved when creating a report:
+
+```ts
+const client = BacktraceClient.initialize(options);
+...
+
+client.addAttribute(() => ({
+    "clientID": resolveCurrentClientId()
+}))
+```
+
+#### Add attributes to an error report
+
+The attributes list of a BacktraceReport object can be directly modified.
+
+```ts
+const report: BacktraceReport = new BacktraceReport('My error message', { myReportKey: 'myValue' });
+report.attributes['myReportKey'] = 'New value';
+```
+
+---
+
+### File Attachments
+
+Files can be attached to error reports. This can be done when initalizing the BacktraceClient, updating the
+BacktraceClient, or dynamically for specific reports. When including attachments in BacktraceClient, all files will be
+uploaded with each report.
+
+```ts
+// Import attachment types from @backtrace/electron
+import { BacktraceStringAttachment, BacktraceUint8ArrayAttachment, BacktraceFileAttachment } from "@backtrace/electron";
+
+// BacktraceStringAttachment should be used for text object like a log file, for example
+const stringAttachment = new BacktraceStringAttachment("logfile.txt", "This is the start of my log")
+
+// Buffer attachment is an attachment type dedicated to store buffer data
+const bufferAttachment = new BacktraceBufferAttachment('buffer-attachment.txt', Buffer.from('sample'));
```

Comment:
Buffer attachment is not present in the import statement

Reply:
Changed.